array-manipulation-1/src/initial.js

```diff
@@ -0,0 +1,15 @@
+/* exported initial */
+//  New list containing all items except the last item
+//  Go through every item in the list
+//    IF the item is not the last item, put it in the list
+//  Give back the new list
+
+function initial(array) {
+  var newArr = [];
+  for (let i = 0; i < array.length; i++) {
+    if (i !== array.length - 1) {
```

Comment:
Instead of having a conditional run on every iteration of this loop, you should just stop the loop one index sooner